base_classes/NXdetector_group.nxdl.xml

```diff
@@ -27,37 +27,55 @@
     name="NXdetector_group" 
 	type="group" extends="NXobject">
 	<doc>
-		Logical grouping of detector elements.
-		
-		This class is used to allow a logical
-		grouping of detector elements (e.g. which tube, bank or group of banks) to be
-		recorded in the file. As well as allowing you to e.g just select the "left" or
-		"east" detectors, it may also be useful for determining which elements belong to the
-		same PSD tube and hence have e.g. the same dead time.
-		
-		For example, if we had "bank1" composed
-		of "tube1", "tube2" and "tube3" then group_names would be the string "bank1,
-		bank1/tube1, bank1/tube2,bank1/tube3" group_index would be {1,2,3,4} group_parent
-		would be {-1,1,1,1}
-		
-		The mapping array is interpreted as
-		group 1 is a top level group containing groups 2, 3 and 4
-		
-		A ``group_index`` array in
-		``NXdetector`` gives the base group for a detector element.
+		Optional logical grouping of detectors. When used, describes 
```

Comment:
MUST be on one line only.

Remove first word `Optional` per discussion this morning.  (Redundant with `When used, ` ...)